e2e/pages/adf/process-services/filtersPage.ts

```diff
@@ -17,13 +17,13 @@
 
 import { by, element } from 'protractor';
 import { Util } from '../../../util/util';
-import { ContentListPage } from '../dialog/contentListPage';
+import { DataTablePage } from '../dataTablePage';
 
 export class FiltersPage {
 
     activeFilter = element(by.css('mat-list-item[class*="active"]'));
     nameColumn = by.css('div[class*="adf-datatable-body"] div[class*="adf-datatable-row"] div[class*="--text"] span');
```

Comment:
This locator should be part of datatable. Just provide the name of the column from here